ansible/roles/os_zabbix/vars/template_app_zabbix_server.yml

```diff
@@ -303,21 +303,6 @@ g_template_app_zabbix_server:
      zabbix_type: trapper
 
    zhttptests:
```

Comment:
I think this needs to be a []

Reply:
Nevermind, I thought this was the only element of the list being deleted. The newlines were throwing me off.